doc/programming_guide/rendering.rst

```diff
@@ -162,21 +162,35 @@ are both contained in the ``WindowBlock`` uniform block. Which looks like this i
 
 You can view what uniform blocks exist in a :py:class:`~pyglet.graphics.shader.ShaderProgram` using the `uniform_blocks`
 property. This is a dictionary containing a Uniform Block name key to a :py:class:`~pyglet.graphics.shader.UniformBlock`
-object value. In the above example, the name would be ``WindowBlock`` while the ``window`` identifier is used in the GLSL
-shader itself.
+object value. In the above example, the name would be ``WindowBlock`` while the ``window`` instance identifier is used in the GLSL shader itself.
+
+Normally with OpenGL, you would have to manually assign a global binding point value to each Uniform Block for each Shader Program, as they are created. With Pyglet, the global binding values and assignments are all taken care of internally.
```

Comment:
Not sure whether the docs should mention `shader_bind_management`. Since it's very specific, and quite a few pyglet options seem to never be referenced in the docs of the regions they influence, might be fine to leave out.

Reply:
I think it's fine if it's mentioned. By default it's True, if someone knows what to do, they will turn it off.  Benjamin and I have discussed this before, we would like to make as many 'hidden' options documented as possible, either for debug purposes or advanced usage.